### PR TITLE
Enhance

### PR DESCRIPTION
**Pull request by jvarness**
@pcarn, wanna look this over and see if it's more up your alley?

### CODE REVIEW

File: src/lunchtime.coffee [hunk 1]
```diff
@@ -40,14 +41,14 @@ queryYelp = (msg, usrLocation, category) ->
...
+  
   signature = oauth.generate('GET', 'https://api.yelp.com/v2/search/', params, CONSUMER_SECRET, TOKEN_SECRET, { encodeSignature: false})
   params.oauth_signature = signature
-
+  
   msg.http('https://api.yelp.com/v2/search/').query(params).get() (err, res, body) ->
     
     if err
-      msg.send 'Error :( {#err}'
+      msg.send 'Error :( #{err}'
```
Comment (pcarn) [inline src/lunchtime.coffee]: Still needs double quotes for string interpolation

**Comment by jvarness** [comment]
Fixed... It's kinda weird that Codacy wouldn't pick something like that up.

**Comment by jvarness** [comment]
For reference, this fixes https://github.com/jvarness/hubot-yelp/issues/2

**Comment by pcarn** [comment]
Looks good. I think something that would make it better would be to let us configure the address of our office, and have it find places close to us, instead of general KC. Here's some code that does that if you want to check it out. https://github.com/github/hubot-scripts/blob/master/src/scripts/yelp.coffee

**Comment by pcarn** [comment]
Also, for the command, I would have the regex say that\'?s so if people use an apostrophe it will still accept it.

**Comment by jvarness** [comment]
I'll consider using the optional apostrophe in a later release.

As for using the address of the campus, I think we can use the `HUBOT_YELP_DEFAULT_LOCATION` setting. We can set that location to the office if we want.

Also, I think what we could add to supplement the script might be allowing us to specify a radius in the configuration. I'm not sure what the default is, but we can discuss this later.

**Pull request merged**